Clean things up (#1)
vulcandth/poke-cia: README.md

> # CIA building extension for Pret Pokemon Gen I / II Repos

Reviewer: ```suggestion
# CIA building extension for Pret Pokémon Gen I / II Repos
```

> This repo provides a simple extension that integrates the build of a Virtual Console .cia file to the pret pokemon Gen I / II repos, to ease the building of a VC .cia for your ROM Hack down to something as simple as typing `make`.

Reviewer: ```suggestion
This repo provides a simple extension that repackages a Nintendo 3DS Virtual Console (VC) `.cia` file using the built `.gbc`(s) and `.patch`(s) generated from the Pret Pokémon Gen I / II repos. This will ease the building of a VC `.cia` for your ROM Hack down to something as simple as typing `make`.
```

## Installation

First, [clone this repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository), and `cd` into it.

Next, you will need to create your `config.mk` file by using `config.mk.template` as a base.

```console
$ cp config.mk.template config.mk
```

Modify this new `config.mk` file using a text editor of your choice.

- Define `roms_names` to match the name of the ROM you want to build the `.cia` from, sans file extension.
  For example, for Pokémon Crystal, you can uncomment one of the example lines:

  ```makefile
  roms_names := pokecrystal11
  ```

  (There should not be more than one uncommented line at a given time.)
  You can also build more than one `.cia` at a time!
  Simply write a space-separated list of names instead:

  ```makefile
  roms_names := redstar bluestar
  ```

- Still in that same file, you must also set the `repo_path` variable to point to the repository containing the ROMs, for example:

  ```makefile
  repo_path  := ../pokered
  ```

  (Relative paths must be relative to the `poke-cia` directory.)

> Copy your original dumped .cia files to `<build_name>.orig.cia`, where `<build_name>` is one of the names you put in `roms_names`.

Reviewer: ```suggestion
Copy and rename your original dumped .cia files to `<build_name>.orig.cia`, where `<build_name>` is one of the names you put in `roms_names`.
```
